dcpy/models/dataset.py

```diff
@@ -29,4 +41,15 @@ class Column(SortedSerializedBase, extra="forbid"):
     data_type: COLUMN_TYPES | None = None
     description: str | None = None
     is_required: bool = True
-    checks: Checks | None = None
+    checks: Checks | list[str | dict[str, CheckAttributes]] | None = (
+        None  # TODO: delete Checks after refactoring metadata
+    )
+
+    @field_validator("checks", mode="after")
+    @classmethod
+    def check_checks(cls, checks: list[str | dict[str, CheckAttributes]]):
+        if isinstance(checks, list):
+            for check in checks:
+                if isinstance(check, dict) and len(check) != 1:
+                    raise ValueError(f"{check} must contain exactly one key-value pair")
```

Comment:
thoughts about just adding a `name` attribute to `CheckAttributes` and storing it as a list, to avoid this type of validation?